src/commcare_cloud/commands/terraform/aws.py

```diff
@@ -598,11 +599,11 @@ def _has_valid_v1_session_credentials(aws_profile):
     if aws_profile not in config.sections():
         return False
     try:
-        expiration = datetime.strptime(config.get(aws_profile, 'expiration'), "%Y-%m-%dT%H:%M:%SZ")
+        expiration = parser.isoparse(config.get(aws_profile, 'expiration'))
     except configparser.NoOptionError:
         return False
 
-    return datetime.utcnow() < expiration
```

Comment:
same question about the comparison

Reply:
See https://github.com/dimagi/commcare-cloud/pull/4455#issuecomment-759717915